sql-files/main.sql

```diff
@@ -286,6 +286,7 @@ CREATE TABLE IF NOT EXISTS `char` (
   `body_direction` tinyint(1) unsigned NOT NULL default '0',
   `disable_call` tinyint(3) unsigned NOT NULL default '0',
   `disable_partyinvite` tinyint(1) unsigned NOT NULL default '0',
+  `disable_showcostumes` tinyint(3) unsigned NOT NULL default '1',
```

Comment:
Just wondering if it's a mistake I didn't check the logic, default 1 ?

Reply:
> Just wondering if it's a mistake I didn't check the logic : default is 1 ?

This button not working like the show equip button, so if you check it, thats mean your costume are showing, uncheck, costume are disabled
disabled=0
enabled=1